src/LegacyCallHandler.tsx

```diff
@@ -119,6 +158,7 @@ export default class LegacyCallHandler extends EventEmitter {
     // call with a different party to this one.
     private transferees = new Map<string, MatrixCall>(); // callId (target) -> call (transferee)
     private audioPromises = new Map<AudioID, Promise<void>>();
+    private audioElementsWithListeners = new Map<HTMLMediaElement, boolean>();
     private supportsPstnProtocol = null;
```

Comment:
The `Typescript Strict Error Checker (--noImplicitAny)` CI job is failing against unrelated adjacent lints, https://github.com/matrix-org/matrix-react-sdk/actions/runs/3580627599/jobs/6022910920

Can we ignore?

Reply:
Asked out-of-band in an [internal room](https://matrix.to/#/!fLeHeojWgBGJlLNdLC:matrix.org/$MRXinLRLZ0J0sxsRUm7jWAIUB2xqnyVGsE4WtKWZB00?via=matrix.org&via=element.io&via=vector.modular.im) and @t3chguy also agreed that these are unrelated ⏩